server/config/disaster_dashboard/Earth.textproto

```diff
@@ -29,6 +29,7 @@ metadata {
       name: "Earthquake"
       event_type_dcids: "EarthquakeEvent"
       color: "#930000"
+      severity_props: "magnitude"
```

Comment:
For my own understanding: Are earthquakes the only event type with magnitudes? The severity filter will eventually be applied to things like droughts and hurricanes, right?

Reply:
I believe Earthquakes are the only event type with magnitudes right now. Yes, the severity filter will be applied for any event types that we set severity_props for in the config